Formula/timidity.rb

```diff
@@ -29,6 +33,7 @@ class Timidity < Formula
   end
 
   def install
+    system "./autogen.sh"
```

Comment:
This should only be on ARM and should be reported upstream

Reply:
I'll change to only run on arm and figure out how to report upstream thank you.

Reply:
Fixed, lmk what you think.
Checking into upstream, it generally takes me some time to get things pushed.